<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GenerateKey">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>GENERATE KEY</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>581</width>
      <height>191</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="FileLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseFile">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>100</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Encrypt">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>150</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Encrypt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Decrypt">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>150</y>
       <width>211</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Decrypt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="BrowseKey">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse Key</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="KeyLine">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>411</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;ENCRYPTION / DECRYPTION APP USING FERNET MODULE&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Designed by Eli ( www.elkoam.site / +233548287700)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
